_data/sites.yml

```diff
@@ -173,11 +173,16 @@
   method: darkonly
   last_checked: 2023-01-04
   
+- domain: kaifa.ch
+  url: https://kaifa.ch/
+  method: darkonly
+  last_checked: 2023-05-23
+  
 - domain: kathut.de
   url: http://kathut.de/
   method: darkonly
   last_checked: 2023-04-16
-
+  
```

Comment:
Could you remove this whitespace? 🙂

Reply:
Don't know how that managed to sneak in removed it.